src/Ledger/EssentialAgda.lagda

```diff
@@ -52,4 +53,4 @@ p24 = record p23 { snd = 4 }
 This results a new record, \AgdaFunction{p24}, which denotes the pair
 \AgdaInductiveConstructor{⦅}~\AgdaNumber{2}~\AgdaInductiveConstructor{,}~\AgdaNumber{4}~\AgdaInductiveConstructor{⦆}.
 \\[6pt]
-See also \href{https://agda.readthedocs.io/en/v2.6.4/language/record-types}{\texttt{https://agda.readthedocs.io/en/v2.6.4/language/record-types}}.
+See also \url{https://agda.readthedocs.io/en/\AgdaVersion/language/record-types.html}.
```

Comment:
Should we add a macro:
```
\NewDocumentCommand{\hrefAgdaDocs}[o]{%
  \url{https://agda.readthedocs.io/en/\AgdaVersion/\IfValueT{#1}{#1}}%
}
```
so that we use it as `\hrefAgdaDocs[language/record-types.html]